README.md

```diff
@@ -45,9 +44,37 @@ PGPASSWORD=tclstairwaypwd psql postgresql://127.0.0.1:5432/tclstairway -U tclsta
 ```
 
 #### Option B: Local Postgres 
-##### Database Configuration
 Set up a local Postgres instance. To set up TCL's required database for unit tests, run the following command, which will create the DB's and users:
 
 ```sh
 psql -f local-dev/local-postgres-init.sql
 ```
+
+### Local testing
+When working on a TCL package, it is often helpful to be able to quickly test out changes
+in the context of a service repo (e.g. `terra-workspace-manager` or `terra-resource-buffer`)
+running a local server.
+
+Gradle makes this very easy with a `mavenLocal` target for publishing and loading packages:
+
+1. Publish from TCL to your machine's local Maven cache.
+   
+   ```
+   ./gradlew publishToMavenLocal
+   ```
+    
+2. From the service repo, add `mavenLocal()` to the _first_ repository location
+build.gradle file (e.g. before `mavenCentral()`.
+
+   ```
+   # terra-workspace-manager/build.gradle
+   
+   repositories {
+     mavenLocal()
+     mavenCentral()
+     ...
+   }
+   ```
+
+That's it! Your service should pick up locally-published changes. If your changes involved bumping 
+a minor version of a TCL package, be careful to update version numbers accordingly.
```

Comment:
How do I un-publish old versions?

Reply:
I think the official process is "Google a bunch until I find a stackoverflow example that seems about right" 😂 

More seriously though, it might be a nice addendum if you do find a clear path. To some extent, since we're only transiently adding `mavenLocal()` to the service repo, it shouldn't be too much of a concern.